### ISSUE TITLE
Dominika Low Standards behaving differently from Vanilla

### ISSUE THREAD

**Issue by Aurum12** [Kitteh6660/Corruption-of-Champions-Mod#1067]
Issue:As a Drider, scenes with Dominka don't activate with Low Standards 
	
Version:1.0.2_mod_1.4
	
Expected behaviour:Can see Dominika scenes with Low Standards with any type of lower body
	
Actual behaviour:Game acts as though Low Standards has not been activated

### Steps to reproduce the issue

1. Activate Low Standards
2. Become a Drider 
3. Attempt to interact with Dominika

**Comment by Kitteh6660** [comment]
Does this still happen in vanilla CoC? If so, it might be one of the few remaining vanilla bugs. Dominika code has been mostly untouched by the mod.

**Comment by Aurum12** [comment]
No, i've checked and it still works in Vanilla, maybe there was a change to
the way Low Standards itself is implemented?
